common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal.cpp

```diff
@@ -1160,6 +1161,7 @@ bool mon_wlan_hal_dwpal::process_dwpal_event(char *buffer, int bufLen, const std
             return false;
         }
 
+        LOG(DEBUG) << "vap_id     : " << VAP;
```

Comment:
Perhaps this is the reason for the build failure?
```suggestion
        LOG(DEBUG) << "vap_id     : " <<std::string(VAP);
```

Reply:
no - I have commented on the cause of failure